project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_users.scss

```diff
@@ -34,7 +34,11 @@
   }
 
   @media (min-width: $screen-sm-min) {
-    .form-control {
+    .form-control,
+    .input-group {
```

Comment:
Weird side-effect on autocomplete fields:

![screen shot 2018-04-26 at 14 58 28](https://user-images.githubusercontent.com/789034/39307176-68fa67c2-4962-11e8-909c-ab51ce4baab0.png)

Reply:
Are you sure you compiled the css and cleared the caches? :)

Reply:
Yes...no...maybe... :(
Upgrade script does this but after a manual try it works.